src/transport/PairingSession.cpp

```diff
@@ -81,7 +78,7 @@ CHIP_ERROR PairingSession::DecodeMRPParametersIfPresent(TLV::Tag expectedTag, TL
     TLV::TLVType containerType = TLV::kTLVType_Structure;
     ReturnErrorOnFailure(tlvReader.EnterContainer(containerType));
 
-    uint16_t tlvElementValue = 0;
+    uint32_t tlvElementValue = 0;
```

Comment:
This PR is OK, but please check your submodule updates did by mistake. Adding comment here because I can't comment on those lines

Reply:
Done.